src/script/calling/CallingRepository.ts

```diff
@@ -695,6 +695,7 @@ export class CallingRepository {
 
     if (Object.keys(missingStreams).length === 0) {
       // we have everything in cache, just return the participant's stream
+      // (FIXME fix for Chrome v73) return navigator.mediaDevices.getUserMedia({audio: true}).then(() => selfParticipant.getMediaStream());
```

Comment:
Should this be commented?

Reply:
Oh shit, this is not supposed to be in this PR. Good catch